pkg/traceability/muleemitter.go

```diff
@@ -4,6 +4,8 @@ import (
 	"encoding/json"
 	"time"
 
+	"github.com/sirupsen/logrus"
+
```

Comment:
As Vivek suggested in prior comment, should  we take advantage of the global logging provided through the sdk log util?